packages/cashier/src/pages/account-transfer/account-transfer.tsx

```diff
@@ -132,7 +132,7 @@ export default connect(({ client, modules }: TRootStore) => ({
     is_loading: modules.cashier.general_store.is_loading,
     is_switching: client.is_switching,
     is_transfer_confirm: modules.cashier.account_transfer.is_transfer_confirm,
-    is_transfer_locked: modules.cashier.general_store.is_transfer_locked,
+    is_transfer_locked: modules.cashier.general_store.is_transfer_locked, // `is_transfer_locked` doesn't exist!
```

Comment:
`is_transfer_locked` doesn’t exist in `general_store`, Not sure if I've missed something or the code was already wrong 🤔

Reply:
This is defined in `account-transfer-store`. I think this needs to be updated from `general_store` to `account_transfer`.

Reply:
@hamid-deriv Yes you are correct 👍🏻 

Should I update it? 🤔 Won’t break anything? 🤔 

How it was working so far? 😅

Reply:
I think now it always returns `false`. I think it is ok to update it. but it's safer to ask it in the channel to be sure. :+1:

Reply:
@hamid-deriv Updated, Thank you 🌹 🙇🏻